  MODPOST Module.symvers
WARNING: modpost: missing MODULE_DESCRIPTION() in drivers/block/swim3.o
WARNING: modpost: missing MODULE_DESCRIPTION() in drivers/macintosh/ans-lcd.o

--- ./drivers/block/swim3.c.vanilla	2025-12-05 17:21:08.026967211 +0100
+++ ./drivers/block/swim3.c	2025-12-05 17:23:47.342867418 +0100
@@ -1290,4 +1290,5 @@
 
 MODULE_LICENSE("GPL");
 MODULE_AUTHOR("Paul Mackerras");
+MODULE_DESCRIPTION("Driver for the SWIM3 floppy controller");
 MODULE_ALIAS_BLOCKDEV_MAJOR(FLOPPY_MAJOR);
--- ./drivers/macintosh/ans-lcd.c.vanilla	2025-12-05 17:21:08.029433606 +0100
+++ ./drivers/macintosh/ans-lcd.c	2025-12-05 17:23:01.521189257 +0100
@@ -203,3 +203,4 @@
 module_init(anslcd_init);
 module_exit(anslcd_exit);
 MODULE_LICENSE("GPL v2");
+MODULE_DESCRIPTION("lcd driver for Apple Network Servers");
